fairseq/models/bart/model.py

```diff
@@ -59,21 +59,25 @@ def add_args(parser):
             choices=utils.get_available_activation_fns(),
             help='activation function to use for pooler layer'
         )
+        parser.add_argument('--do-r4f', action='store_true', 
```

Comment:
Please rename this to something more descriptive, something like `--spectral-norm-classification-head`

Reply:
Updated.